File: Backend/NoteMaker.API/NoteMaker.API.Model/NoteModel.cs
using System;

namespace NoteMaker.API.Model
{
    public class NoteModel
    {
        public int ID { get; set; }
        public string NoteTitle { get; set; }
        public string NoteContent { get; set; }
        public bool Starred { get; set; } = false;
        public DateTime CreatedTime { get; set; } = DateTime.Now;
        public DateTime ModifiedTime { get; set; } = DateTime.Now;
    }
}

File: Frontend/note-maker-web/src/models/NoteModel.ts
export class NoteModel 
{
    id: number;
    noteTitle: string;
    noteContent: string;
    starred: boolean;
    createdTime: Date;
    modifiedTime: Date;
    
    constructor(note: Partial<NoteModel>)
    {
        Object.assign(this, note);
    }
}